src/RageFileManager.cpp

```diff
@@ -432,6 +432,22 @@ void RageFileManager::GetDirListing( const RString &sPath_, vector<RString> &Add
 
 	UnreferenceAllDrivers( apDriverList );
 
+	// Remove files that start with ._ from the list because these are special
+	// OS X files that cause interference on other platforms. -Kyz
+	for(size_t i= iOldSize; i < AddTo.size(); ++i)
```

Comment:
There is possibly another item to add to the exclusion: any directory that contains `__MACOSX`. However, the only files in that folder are typically the ones being excluded below. It may be better to be safe and exclude that as well, but it was not necessary for this round of testing.